src/main/java/io/mvnpm/esbuild/Execute.java

```diff
@@ -66,7 +66,7 @@ public Process execute(BuildEventListener listener) throws IOException {
     private String[] getCommand() {
         String[] command = args != null ? getCommand(args) : getCommand(esBuildConfig);
         if (logger.isLoggable(Level.FINEST)) {
-            logger.log(Level.FINEST, "running esbuild with flags: `{0}`.", String.join(" ", command));
+            logger.log(Level.FINEST, "running esbuild with flags: \n > {0}`.", String.join(" ", command));
```

Comment:
Here you only have one ` . Is that correct ?

Reply:
oupsy